.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+      - alpha
+      - beta
+      - next
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+      - name: Install pnpm and dependencies
+        uses: pnpm/action-setup@v2
+        with:
+          version: 8
+          run_install: true
+      - name: Build
+        run: pnpm build:release
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
```

Comment:
We'll need to install semantic-release and its plugins to complete this job, like specified in our [Automate releases guide](https://github.com/warp-ds/release-playground/wiki/Automate-releases-with-semantic-release-and-Eik).

Reply:
Moved changes related to automated releases to https://github.com/warp-ds/icons/pull/7